File: python/problem1012.py
entry = input();

list = entry.split(' ');

A = float(list[0]);
B = float(list[1]);
C = float(list[2]);
pi = 3.14159;

triangle = (A * C)/2;
circle = pi * C * C;
trapezium = ((A + B) * C)/2;
square = B * B;
rectangle = A * B;

print("TRIANGULO: %.3f" % triangle);
print("CIRCULO: %.3f" % circle);
print("TRAPEZIO: %.3f" % trapezium);
print("QUADRADO: %.3f" % square);
print("RETANGULO: %.3f" % rectangle);

File: java/Problem1012.java
import java.util.Scanner;

public class Problem1012 {
	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);
		String[] x = sc.nextLine().split(" ");
		double a = Double.parseDouble(x[0]);
		double b = Double.parseDouble(x[1]);
		double c = Double.parseDouble(x[2]);
		System.out.printf("TRIANGULO: %.3f\n", (a * c)/2);
		System.out.printf("CIRCULO: %.3f\n", 3.14159 * (c * c));
		System.out.printf("TRAPEZIO: %.3f\n", ((a + b) * c)/2);
		System.out.printf("QUADRADO: %.3f\n", b * b);
		System.out.printf("RETANGULO: %.3f\n", a * b);
		sc.close();
	}
}
